src/ngraph/file_util.hpp

```diff
@@ -95,5 +97,37 @@ namespace ngraph
         /// \param path The path to test
         /// \return true if the path exists, false otherwise
         bool exists(const std::string& path);
+
+        /// \brief      Reads a binary file to a vector.
+        ///
+        /// \param[in]  path  The path where the file is located.
+        ///
+        /// \tparam     T     The type we want to interpret as the elements in binary file.
+        ///
+        /// \return     Return vector of data read from input binary file.
+        ///
+        template <typename T>
+        std::vector<T> read_binary_file(const std::string& path)
```

Comment:
If we're not using the `read_binary_file` below, let's remove the move of the file from the PR.